bin/python_scripts/find_invalid_tags.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Set up environment from ckan.ini
+# export POSTGRES_URL=<sqlalchemy.url from ckan.ini>
+#
+# Execute script like this -
+# python find_invalid_tags.py
+#
+
+import io
+import os
+import sys
+import psycopg2
+import subprocess
+
+import logging
+
+from ckan.logic.schema import default_tags_schema
+from ckan.lib.navl.dictization_functions import validate, Invalid
+
+POSTGRES_URL = os.environ.get('POSTGRES_URL')
```

Comment:
```suggestion
POSTGRES_URL = os.environ['POSTGRES_URL']
```

I'm guessing most of this script will fail if it's not set?

Reply:
Yes it will fail, I guess better to fail early